Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -181,9 +181,10 @@ public struct WorldScaleSceneView: View {
         .onCameraTrackingStateChanged { trackingState in
             onCameraTrackingStateChangedAction?(trackingState)
         }
-        .onTapGesture { tapPoint in
-            handleSingleTap(tapPoint)
-        }
+        // Workaround for `onTapGesture` Apple bug #FB20686694.
+        .simultaneousGesture(SpatialTapGesture().onEnded({ event in
+            handleSingleTap(event.location)
+        }))
```

Comment:
You can avoid one set of brackets.
```suggestion
        .simultaneousGesture(SpatialTapGesture().onEnded { event in
            handleSingleTap(event.location)
        })
```